libs/Model/Model.cpp

```diff
@@ -52,16 +50,20 @@ void Model::render(ShaderProgram& shaderProgram)
 
 	assert(m_Obj_Model && "NO MODEL TO RENDER");
 	// bind texture
-	for (int i = 0; i < m_textures.size(); ++i)
-	{
-		m_textures[i]->Bind(i);
-	}
+	
+	m_texture->Bind(0);
+	
 
 	//render geometry
 	if (m_Obj_Model)
 	{
 		m_Obj_Model->render();
 	}
+
+	//Unbind Text
+  
+  m_texture->UnBind(0);
```

Comment:
formatting/alignment is off in this function

Reply:
I formatted the file